apps/admin-portal/src/locales/en/views.ts

```diff
@@ -20,6 +20,79 @@ import { Views } from "../../models";
 
 export const views: Views = {
     components: {
+        applications: {
+            notifications: {
+                fetchApplications: {
+                    error: {
+                        description: "{{description}}",
+                        message: "Error retrieving applications"
+                    },
+                    genericError: {
+                        description: "Couldn't retrieve applications",
+                        message: "Something went wrong"
+                    },
+                    success: {
+                        description: "Successfully retrieved the applications.",
+                        message: "Applications retrieval successful"
+                    }
+                }
+            },
+            placeholders: {
+                emptyList: {
+                    action: "Refresh list",
+                    subtitles: {
+                        0: "The applications list returned empty.",
+                        1: "This could be due to having no discoverable applications.",
+                        2: "Please ask an admin to enable discoverability for applications."
+                    },
+                    title: "No Applications"
+                }
+            },
+            search: {
+                forms: {
+                    searchForm: {
+                        inputs: {
+                            filerAttribute: {
+                                label: "Filter attribute",
+                                placeholder: "ex: name, description etc.",
```

Comment:
```suggestion
                                placeholder: "E.g. name, description etc ...",
```

Do it in other places as well.

Reply:
Fixed in `d9385d4`